backend/benefit/users/api/v1/views.py

```diff
@@ -15,7 +13,6 @@
 from users.api.v1.authentications import HelsinkiProfileApiTokenAuthentication
 from users.api.v1.permissions import BFGDPRScopesPermission
 from users.api.v1.serializers import UserSerializer
-from users.utils import set_mock_user_name
```

Comment:
It seems `set_mock_user_name` is dead code after this change so it could be removed IMHO as clean up.

Reply:
Ok, then I will remove it.